src/components/App.js

```diff
@@ -78,10 +78,7 @@ class App extends Component {
       };
       return shift(Math.round(shift(number, precision, false)), precision, true);
     };
-    let rounded = round(this.state.data[0]['Total Review Stars'] / this.state.data[0]['Review Total'], 1).toString();
```

Comment:
Do you think there's a way you could use variable names to make this more readable to other people on the team?